Issue/263 remove composition apps

warpech opened the pull request:
Implements #263.

@tomalec do you think that some description should be added to README.md, about where to find CompositionProvider and CompositionEditor?

tomalec commented:
> do you think that some description should be added to README.md, about where to find CompositionProvider and CompositionEditor?

I think so, I'd also add it to release notes. At least for CompositionProvider - to make other apps running keep the same look and feel.

other than that, LGTM

warpech commented:
I have removed the irrelevant information from [README.md](https://github.com/StarcounterPrefabs/Launcher/blob/3cb812f5376d6eb8235461c6036e5ffe5b1504e7/README.md), added some info about where to find CompositionProvider and CompositionEditor and added information about Workspaces (https://github.com/Starcounter/TechComm/issues/213).
